test/csuite/timestamp_abort/main.c

```diff
@@ -97,10 +97,80 @@ usage(void)
 typedef struct {
 	WT_CONNECTION *conn;
 	uint64_t start;
-	uint32_t id;
+	uint32_t info;
 } WT_THREAD_DATA;
 
 /*
+ * thread_ts_run --
+ *	Runner function for a timestamp thread.
+ */
+static WT_THREAD_RET
+thread_ts_run(void *arg)
+{
+	WT_CURSOR *cur_stable;
+	WT_SESSION *session;
+	WT_THREAD_DATA *td;
+	uint64_t i, last_ts, oldest_ts;
+	int ret;
+	char tscfg[64];
+
+	td = (WT_THREAD_DATA *)arg;
+	last_ts = 0;
+
+	if ((ret = td->conn->open_session(td->conn, NULL, NULL, &session)) != 0)
+		testutil_die(ret, "WT_CONNECTION:open_session");
+	if ((ret = session->open_cursor(
+	    session, stable_store, NULL, NULL, &cur_stable)) != 0)
+		testutil_die(ret, "WT_SESSION.open_cursor: %s", stable_store);
+
+	/*
+	 * Every N records we will record our stable timestamp into the
+	 * stable table.  That will define our threshold where we
+	 * expect to find records after recovery.
+	 */
+	for (;;) {
+		oldest_ts = UINT64_MAX;
+		/*
+		 * For the timestamp thread, the info field contains the
+		 * number of worker threads.
+		 */
+		for (i = 0; i < td->info; ++i) {
+			/*
+			 * We need to let all threads get started, so if we
+			 * find any thread still with a zero timestamp we
+			 * go to sleep.
+			 */
+			if (th_ts[i] == 0)
+				goto ts_wait;
+			if (th_ts[i] != 0 && th_ts[i] < oldest_ts)
+				oldest_ts = th_ts[i];
+		}
+
+		if (oldest_ts != UINT64_MAX &&
+		    oldest_ts - last_ts >  STABLE_PERIOD) {
```

Comment:
Extra space after the operator.

Reply:
You don't really need `last_ts`, you could compare against `global_ts`, but *shrug*.